src/components/Actions.tsx

```diff
@@ -62,7 +65,8 @@ export function SelectedShapeActions({
       <fieldset>
         <legend>Layers Actions</legend>
```

Comment:
Can you add a translation for this one?

Reply:
Just "Actions" maybe: https://github.com/excalidraw/excalidraw/pull/1140/files#r401100390